code.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+#Importing dataset from 'data.txt' delimited by tab
+matrix=np.genfromtxt('data.txt',delimiter='	')
```

Comment:
Abhishek you're not loading the second label from the data at all. check whether your data is getting loaded properly or not. Add a print statement after the label 2 loading code so you can see. Check the dimensions of a and b vectors after you are done with the loops.